#Repairnator properties
#Tue Mar 28 05:26:05 CEST 2017
step-durations=[CheckoutBuild\=5, \=4, ComputeClasspath\=18, CloneRepository\=36, NopolRepair\=95, PushIncriminatedBuild\=55, ComputeSourceDir\=0, SquashRepository\=42]
workspace=./workspace
pr-remote-repo=jihoonson/druid
pr-head-commit-id=05a7d63205fafda750d614f7b94480764d57ab97
pr-id=4077
repo=druid-io/druid
failingModule=/root/workspace/druid-io/druid/215752779/server
is-pr=true
buildid=215752779
error-types=java.lang.AssertionError
failing-test-cases=FailureLocation{className\='io.druid.segment.loading.SegmentLoaderLocalCacheManagerTest', failingMethods\=[io.druid.segment.loading.SegmentLoaderLocalCacheManagerTest\#testRetryAllFail, io.druid.segment.loading.SegmentLoaderLocalCacheManagerTest\#testRetrySuccessAtSecondLocation], erroringMethods\=[], failures\=[FailureType{failureName\='java.lang.AssertionError', failureDetail\='null', isError\=false}, FailureType{failureName\='java.lang.AssertionError', failureDetail\='null', isError\=false}], nbFailures\=2, nbErrors\=0},FailureLocation{className\='io.druid.segment.loading.LocalDataSegmentPusherTest', failingMethods\=[io.druid.segment.loading.LocalDataSegmentPusherTest\#testPushCannotCreateDirectory], erroringMethods\=[], failures\=[FailureType{failureName\='java.lang.AssertionError', failureDetail\='Expected test to throw (an instance of java.io.IOException and exception with message a string containing "Unable to create directory")', isError\=false}], nbFailures\=1, nbErrors\=0},FailureLocation{className\='io.druid.query.lookup.LookupSnapshotTakerTest', failingMethods\=[io.druid.query.lookup.LookupSnapshotTakerTest\#testIOExceptionDuringLookupPersist], erroringMethods\=[], failures\=[FailureType{failureName\='java.lang.AssertionError', failureDetail\='Expected exception\: io.druid.java.util.common.ISE', isError\=false}], nbFailures\=1, nbErrors\=0}
pr-base-commit-id=3ec1877887b3cd3b9ff6ea95a62bff29162b752a
